tests/utils/datavolume.go

```diff
@@ -40,6 +40,8 @@ const (
 	CirrosURL = "http://cdi-file-host.%s/cirros-qcow2.img"
 	// ImageioURL provides URL of oVirt engine hosting imageio
 	ImageioURL = "https://imageio.%s:12346/ovirt-engine/api"
+	// NoImage404URL provides a fake broken url - a 404
+	NoImage404URL = "http://cdi-file-host:82/nonexistent.qcow2"
```

Comment:
We already have a constant with an invalid url: InvalidQcowImagesURL, please use that instead of making a new one.

Reply:
good catch, I have missed that,